Tasks/Common/java-common/module.json

```diff
@@ -1,6 +1,6 @@
 {
   "messages": {
     "LocateJVMBasedOnVersionAndArch": "Locate JAVA_HOME for Java %s %s",
-    "FailedToLocateSpecifiedJVM": "Failed to find specified JDK version. Please make sure environment variable '%s' exists and is set to the location of a corresponding JDK."
+    "FailedToLocateSpecifiedJVM": "Failed to find specified JDK version. Please ensure specified JDK version is installed on the agent or use the JavaToolInstaller task to install the desired JDK. Refer to https://docs.microsoft.com/en-us/vsts/build-release/tasks/tool/java-tool-installer for more details."
```

Comment:
Instead of including the URL in the text, can we turn it into a hyperlink behind other text?  We could remove the last sentence and change this sentence to have the hyperlink using markdown:

```
Please ensure specified JDK version is installed on the agent or use the [Java Tool Installer](https://docs.microsoft.com/en-us/vsts/build-release/tasks/tool/java-tool-installer) task to install the desired JDK.
```